client/app/services/authentication/authentication.service.js

```diff
@@ -21,6 +21,12 @@ class AuthenticationService {
       .catch((res) => this.$q.reject(res.data));
   }
 
+  verify(userEmail) {
+    return this.$http.post("/api/users/resend_verification/", { email: userEmail })
```

Comment:
You don't need to provide the users' mail or any other data. (You can also remove that part in the controller)